src/common/Error/index.js

```diff
@@ -0,0 +1,15 @@
+import { Header, Paragraph, ErrorWrapper } from "./styled";
+import { ReactComponent as WarningIcon } from "../../assets/error.svg";
+import { ButtonLink } from "./ButtonLink";
+
+export const Error = () => (
+  <ErrorWrapper>
+    <WarningIcon />
+    <Header>Oooops! Something went&nbsp;wrong...</Header>
+    <Paragraph>
+      Please, check your network connection <br />
+      and try again
+    </Paragraph>
+    <ButtonLink to="/movies">Back to Homepage</ButtonLink>
```

Comment:
In figma there is: home page. For me that's look better, but it can be discussed.

Reply:
exactly Home  page is better - I will change it.